docs/source/_html/sitemap-override.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
THe only issue is that we use the getdaft.io domain for other things too besides the docs https://www.getdaft.io/
the current sitemap in this PR would prevent that stuff from getting indexed.

Reply:
Just added: https://github.com/Eventual-Inc/Daft/pull/1752/files#diff-6fc1b44a65e12f55e44094212bd9f3ffbe5391cad422d494e43dea2fad3983e6R14-R18